src/geo/map/layer-types.js

```diff
@@ -0,0 +1,36 @@
+var TILED_LAYER_TYPE = 'Tiled';
```

Comment:
I feel we should make use of this in Builder instead of having another similar to this there.